client/scripts/BFOUNDATION.as

```diff
@@ -3170,7 +3170,7 @@ package
             {
                _loc1_.r4over = true;
             }
-            _loc1_.time = _loc2_.time;
+            _loc1_.time.Set(_loc2_.time.Get());
```

Comment:
Just a question for my own understanding - why do we remove the assignment here and directly call `.Set()` on the time property?

Could this work:

```javascript
_loc1_.time = new SecNum(_loc2_.time);
````